src/calculator/types.ts

```diff
@@ -161,9 +171,22 @@ export namespace EAC {
 
     const OptionsDetails = Record({
         exerciseCostUSD: Number,
-        grossProceedsUSD: Number,
-        netProceedsUSD: Number,
+        grossProceedsUSD: Number.optional(),
+        netProceedsUSD: Number.optional(),
+    });
+
+    const ExerciseAndHoldTransaction = Record({
+        action: Literal(Action.ExerciseAndHold),
+        date: InstanceOf(Date),
+        symbol: String,
+        description: String,
+        quantity: Number,
+        feesUSD: Number.optional(),
+        amountUSD: Number.optional(),
```

Comment:
Are these fields precent only on some Exercise and Hold transactions? Why some have these and others don't?